crates/nomination/src/ext.rs

```diff
@@ -31,50 +31,9 @@ pub(crate) mod vault_registry {
         <vault_registry::Pallet<T>>::get_backing_collateral(vault_id)
     }
 
-    pub fn liquidate_vault_with_status<T: vault_registry::Config>(
-        vault_id: &T::AccountId,
-        status: VaultStatus,
-    ) -> Result<Backing<T>, DispatchError> {
-        <vault_registry::Pallet<T>>::liquidate_vault_with_status(vault_id, status)
-    }
-
-    pub fn increase_backing_collateral<T: vault_registry::Config>(
-        vault_id: &T::AccountId,
-        amount: Backing<T>,
-    ) -> DispatchResult {
-        <vault_registry::Pallet<T>>::try_increase_backing_collateral(vault_id, amount)
-    }
```

Comment:
I think not all of these functions need to be removed but I can re-add them after the slashing is implemented. Perhaps the way backing collateral is increased will change with the new fee optimizations?

Reply:
Yes there will now be two traits (`TryDepositCollateral` and `TryWithdrawCollateral`) that you would implement.